packages/atomic/template/stencil.config.ts

```diff
@@ -31,7 +32,13 @@ export const config: Config = {
   devServer: {
     port: parseInt(process.env.STENCIL_PORT) || 3333,
   },
-  plugins: [dotenvPlugin()],
+  plugins: [
+    replace({
+      'process.env.PLATFORM_URL': `'${process.env.PLATFORM_URL}'`,
+      'process.env.ORGANIZATION_ID': `'${process.env.ORGANIZATION_ID}'`,
```

Comment:
These 2 variables are needed for the Stencil app (the other variables are needed by the lambda function only)